File: src/main/java/frc/team5190/feeder/Feeder.java
package frc.team5190.feeder;

import edu.wpi.first.wpilibj.networktables.NetworkTable;

public class Feeder {

    public static void main(String[] args) {
        NetworkTable.setClientMode();
        NetworkTable.setTeam(5190);
        NetworkTable.setIPAddress("10.51.90.2");
        NetworkTable.initialize();
        new PathFeeder();
        new VisionFeeder(false);
    }
}

File: src/main/kotlin/frc/team5190/feeder/Feeder.kt
package frc.team5190.feeder

import edu.wpi.first.networktables.NetworkTableInstance

object Feeder {

    @JvmStatic
    fun main(args: Array<String>) {
        with(NetworkTableInstance.getDefault()) {
            startClientTeam(5190)
            setUpdateRate(0.02)
        }
        /* NetworkTable.setClientMode()
         NetworkTable.setTeam(5190)
         NetworkTable.setIPAddress("10.51.90.2")
         NetworkTable.initialize()*/
        PathFeeder
        VisionFeeder(false)
    }
}